src/applications/personalization/profile/components/direct-deposit/BankInfo.jsx

```diff
@@ -287,7 +288,11 @@ export const BankInfo = ({
           >
             {/* eslint-disable jsx-a11y/no-redundant-roles */}
             <img
-              src="/img/direct-deposit-check-guide.svg"
+              src={
```

Comment:
why replace an SVG with a PNG? I did just notice that for some reason the current SVG does not seem to exist on staging, but it does show up locally. So that's something that needs to be looked into; I swear it used to work since the SVG was approved in staging months ago.

Reply:
I used a PNG because in the [original story](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/12343) it was recommended in order to get a higher resolution. I can use the SVG if thats is preferred format.

Reply:
From [my comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12343#issuecomment-940526865) on that issue:

> You may be able to get a higher resolution png by creating an export from the svg file.

It doesn't look like there are any comments there recommending using PNGs over SVGs for their resolution. Let me know if I'm overlooking something.

Reply:
Okay, is the SVG preferred then?